### ISSUE TITLE
Select Multiple Tests to Run

### ISSUE THREAD

**Issue by dsanders11** [dsanders11/vscode-electron-build-tools#13]
Add a context menu/inline for selecting tests to run so you can do multiple tests in one go. Would need to figure out the UI to make it visually clear to users (ability to add colors would be nice), maybe a way to filter only the selected tests. Probably would need to do multiple runs of `spec-runner.js` to avoid command line length issues, and the hairiness of a super big regex for the selected tests. Could possibly be done in parallel.

**Comment by dsanders11** [comment]
The UI for this is supported thanks to the VS Code testing view, so just need to better support it in the test runner - do separate test runs instead of regex to avoid command-line length.

**Comment by dsanders11** [comment]
Changing this to a bug (low priority) to track that this works, but you'll run into a failure at an upper limit at some point if you try to select too many different tests to run. Since this is a niche use case, not worth putting much effort into fixing, but tracking it here.
